components/Footer.tsx

```diff
@@ -3,20 +3,60 @@ import { env } from "@/env.mjs";
 
 export default function Footer() {
   return (
-    <footer className="">
-      <div className="h-full border-t border-secondary-300 bg-secondary-300 p-4 dark:border-secondary-700 dark:bg-secondary-800 lg:p-10">
-        <div className="max-w-5xl text-center text-sm font-bold text-foreground dark:text-primary-500 lg:mx-auto lg:leading-tight">
-          <div className="flex w-full items-center justify-center">
-            Powered by{" "}
-            <span className="ml-4 w-20 brightness-0 dark:brightness-150">
-              <Image
-                src={env.NEXT_PUBLIC_ORG_LOGO as string}
-                alt={env.NEXT_PUBLIC_ORG_NAME as string}
-                width="80"
-                height="22"
-              />
-            </span>
-          </div>
+    <footer className="border-t border-secondary-300 bg-secondary-300 p-4 dark:border-secondary-700 dark:bg-secondary-800 lg:p-8">
+      <div className="mx-auto max-w-4xl text-center text-sm font-medium text-foreground dark:text-primary-500 lg:leading-snug">
+        <div className="mb-4 flex items-center justify-center">
+          Powered by{" "}
+          <span className="ml-4 w-28 brightness-0 dark:brightness-150">
+            <Image
+              src={env.NEXT_PUBLIC_ORG_LOGO as string}
+              alt={env.NEXT_PUBLIC_ORG_NAME as string}
+              width={140}
+              height={36}
+            />
+          </span>
+        </div>
+        <div className="mb-4">
+          <a
+            href={env.NEXT_PUBLIC_DATA_SOURCE}
+            className="mb-1 block font-bold text-blue-600 hover:underline dark:text-blue-400"
```

Comment:
Also, do not hard code specific colours. Different organizations would be having different theme configurations, and may not match with with theme.

Reply:
sure. I will take care of this